portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,23 +38,45 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    comments.clear();
```

Comment:
Rather than having to clear() the list on every doGet(), we can make comment a local variable instead of a class variable.